foundations/html_css/project/html-css-foundations-project.md

```diff
@@ -20,16 +20,16 @@ As we did in the Recipe project, go ahead and set up a git repository before beg
 
 ### Warning About Looking at Other People's Finished Project Code Before You have Completed Your Own
 
-All of our projects have a "Student Solutions" section at the bottom for people to post their completed code and live websites for the projects. These are here for you to compare your **completed** code to and see what other routes people have chosen to take with their project code and designs. 
+All of our projects have a "Student Solutions" section at the bottom for people to post their completed code and live websites for the projects. These are here for you to compare your **completed** code to, and see what other routes people have chosen to take with their project code and designs. 
```

Comment:
Is this comma not a comma splice?

Reply:
It isn't. A comma splice would have two independent clauses on either side of it. This only has an independent clause before the comma. However I think it might not be the best way to break up the sentence, as it does sound kind of clunky. Maybe something like "These are here for you both to compare your **completed** code with and see what other routes people have chosen to take with their project code and designs." How does that sound?

Reply:
Yeah I think that's better! The comma is awkward.